src/js/components/Scope/DatasetScopePicker.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { List, Avatar, Space, Typography } from 'antd';
+import { useTranslationCustom, useTranslationDefault } from '@/hooks';
+import { Link, useLocation } from 'react-router-dom';
+import { FaDatabase } from 'react-icons/fa';
+import { getCurrentPage } from '@/utils/router';
+import type { Project } from '@/types/metadata';
+
+type DatasetScopePickerProps = {
+  parentProject: Project;
+  isSingleProject?: boolean;
+};
+export const DatasetScopePicker = ({ parentProject, isSingleProject }: DatasetScopePickerProps) => {
+  const td = useTranslationDefault();
+  const t = useTranslationCustom();
+  const location = useLocation();
+  const baseURL = '/' + location.pathname.split('/')[1];
+  const page = getCurrentPage();
+  return (
+    <Space direction="vertical" style={{ display: 'flex' }}>
+      <Space align="baseline" size="large">
+        <Typography.Title level={4} className="no-margin-top">
+          {td('Project')}: {t(parentProject.title)}
+        </Typography.Title>
+        {isSingleProject ? (
```

Comment:
in single project mode, clearing it just returns it to the project again since if there's only 1 project it's selected by default. I'd opt for just not showing it here.

Reply:
I added it to give a way to clear the dataset selection in single-project-mode.
Maybe this should clear the dataset only instead, and be there even when we have multiple projects?

Reply:
Narrow scope selection was not automatic, just added it. Project scope is fixed if there is only one project, which disables project selection/clearing.